tests/integration/test_fim/test_stats_integrity_sync/test_FIM_performance.py

```diff
@@ -0,0 +1,631 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import re
+import shutil
+import subprocess
+import time
+from copy import deepcopy
+from enum import Enum
+from multiprocessing import Process, Manager
+
+import pandas as pd
+import pytest
+
+from wazuh_testing import logger
+from wazuh_testing.fim import callback_realtime_added_directory
+from wazuh_testing.tools import WAZUH_PATH, WAZUH_CONF, LOG_FILE_PATH
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.services import control_service, check_daemon_status
+
+root_dir = '/test'
+tested_daemon = 'ossec-syscheckd'
+state_collector_time = 1
+state_path = os.path.join(WAZUH_PATH, 'var', 'run', 'ossec-agentd.state')
+performance_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'stats', 'performance')
+fim_db_path = os.path.join(WAZUH_PATH, 'queue', 'fim', 'db')
+state_configuration = {
+    "agentd.state_interval": state_collector_time,
+    "syscheck.debug": 2
+}
+
+
+class Types(Enum):
+    added = 'added'
+    modified = 'modified'
+    deleted = 'deleted'
+
+
+# Fixtures
+
+@pytest.fixture(scope='module')
+def initial_clean():
+    """Clean the environment."""
+    clean_environment(stats=True)
+
+
+@pytest.fixture(scope='module')
+def modify_local_internal_options():
+    """Replace the local_internal_options file"""
+    with open(os.path.join(WAZUH_PATH, 'etc', 'local_internal_options.conf'), 'w') as f:
+        for conf, value in state_configuration.items():
+            f.write(f'{conf}={value}\n')
+
+
+# Functions
+
+
+def replace_conf(sync_eps, fim_eps):
+    """Configure syscheck in realtime=yes."""
+    directories_regex = r"<directories realtime=\"yes\">[\n\t ]*(TESTING_DIRECTORY)[\n\t ]*</directories>"
```

Comment:
Refactor this block using the configuration utilities in wazuh_testing.

Reply:
We can not use the wazuh_testing configuration utilities because the files and paths are generated at execution of every tests.